### PR TITLE
Create recipe for topic mapping binary

### PR DESCRIPTION
**Pull request by TnTo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by TnTo** [comment]
@conda-forge/staged-recipes

**Comment by TnTo** [comment]
@conda-forge-admin, please ping team

**Comment by TnTo** [comment]
@conda-forge-admin, please ping team

**Comment by TnTo** [comment]
@conda-forge-admin, please ping team

File: recipes/topicmapping/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package:
+  name: topicmapping
+  version: 1.0.0
+
+source:
+  - url: https://bitbucket.org/andrealanci/topicmapping/get/13bb57316ff0.zip
```
Comment (ocefpaf) [inline recipes/topicmapping/meta.yaml:6]: Is that a specific hash? Can we get it from a stable release?

Comment (TnTo) [reply recipes/topicmapping/meta.yaml:6]: This is the only and stable release.

It is the code written for a research paper ( https://arxiv.org/abs/1402.0422 ) and it is not longer developed (it is possible that previous commits had been lost in a change of repository)

This package would be useful to manage dependencies for future research projects.

Comment (ocefpaf) [reply recipes/topicmapping/meta.yaml]: They don't really have tags but it seems that the paper calls it "v1" so let's go with that.

**Pull request merged**